.gitmodules

```diff
@@ -1,14 +1,16 @@
-[submodule "contracts/lib/forge-std"]
-	path = contracts/lib/forge-std
-	url = https://github.com/foundry-rs/forge-std
 [submodule "lib/forge-std"]
-	branch = v1.3.0
-[submodule "contracts/lib/openzeppelin-contracts"]
-	path = contracts/lib/openzeppelin-contracts
-	url = https://github.com/OpenZeppelin/openzeppelin-contracts
-[submodule "contracts/lib/subnet-evm"]
-	path = contracts/lib/subnet-evm
-	url = https://github.com/ava-labs/subnet-evm
-[submodule "contracts/lib/openzeppelin-contracts-upgradeable"]
-	path = contracts/lib/openzeppelin-contracts-upgradeable
+	branch = v1
+	path = lib/forge-std
+	url = https://github.com/foundry-rs/forge-std
+	ignore = dirty
+[submodule "lib/openzeppelin-contracts"]
+	path = lib/openzeppelin-contracts
+	url = https://github.com/openzeppelin/openzeppelin-contracts
+	ignore = dirty
+[submodule "lib/openzeppelin-contracts-upgradeable"]
+	path = lib/openzeppelin-contracts-upgradeable
 	url = https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable
+	ignore = dirty
```

Comment:
do we want ignore=dirty? IIUC if I make local changes to these submodules, git will not tell me in git status, but will it still build with these local changes without me knowing?

Also how come for subnet-evm there isn't the ignore tag

Reply:
Yeah, on further consideration masking local changes would likely cause more headaches than it's worth. Unintended local changes were very common when putting together this PR since the submodules themselves were moving around, but for more regular changes that should not be the case. Removed now.